pkg/router/gateway_api_test.go

```diff
@@ -276,79 +278,39 @@ func TestGatewayAPIRouter_Routes(t *testing.T) {
 	})
 }
 
-func TestGatewayAPIRouter_getSessionAffinityRouteRules(t *testing.T) {
+func TestGatewayAPIRouter_makeFilters(t *testing.T) {
```

Comment:
why remove this test? we can add another test for `makeFilters`.

Reply:
sorry, it's a mistake, i will add old test back